src/script/interpreter.h

```diff
@@ -174,7 +174,34 @@ class MutableTransactionSignatureChecker : public TransactionSignatureChecker
     MutableTransactionSignatureChecker(const CMutableTransaction* txToIn, unsigned int nInIn, const CAmount& amountIn) : TransactionSignatureChecker(&txTo, nInIn, amountIn), txTo(*txToIn) {}
 };
 
-bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript& script, unsigned int flags, const BaseSignatureChecker& checker, SigVersion sigversion, ScriptError* error = NULL);
+class ScriptExecution {
+public:
+    typedef std::vector<unsigned char> StackElementType;
+    typedef std::vector<StackElementType> StackType;
+
+    const CScript& script;
```

Comment:
When that happens either make it public, create a getter or an `Inspect` method?